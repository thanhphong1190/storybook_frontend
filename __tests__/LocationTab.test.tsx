import {
  LocationTab,
  LocationTabProps
} from '../src/components/tabs/LocationTab'
import renderer from 'react-test-renderer'

const props: LocationTabProps = {
  tabItems: [
    { label: 'ACT', url: '#' },
    { label: 'NSW', url: '#' },
    { label: 'QLD', url: '#' },
    { label: 'SA', url: '#' },
    { label: 'VIC', url: '#' },
    { label: 'WA', url: '#' }
  ]
}
it('renders correctly', () => {
  const tree = renderer.create(<LocationTab {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
