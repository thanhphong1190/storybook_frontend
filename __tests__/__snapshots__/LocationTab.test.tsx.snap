// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="LocationTabstyled__LocationTabWrapper-sc-1lchkd7-0 fhnkrs"
>
  <a
    className="LocationTabstyled__LocationTabItem-sc-1lchkd7-1 ewjASn"
    onClick={[Function]}
  >
    ACT
  </a>
  <a
    className="LocationTabstyled__LocationTabItem-sc-1lchkd7-1 ewjASn"
    onClick={[Function]}
  >
    NSW
  </a>
  <a
    className="LocationTabstyled__LocationTabItem-sc-1lchkd7-1 ewjASn"
    onClick={[Function]}
  >
    QLD
  </a>
  <a
    className="LocationTabstyled__LocationTabItem-sc-1lchkd7-1 ewjASn"
    onClick={[Function]}
  >
    SA
  </a>
  <a
    className="LocationTabstyled__LocationTabItem-sc-1lchkd7-1 ewjASn"
    onClick={[Function]}
  >
    VIC
  </a>
  <a
    className="LocationTabstyled__LocationTabItem-sc-1lchkd7-1 ewjASn"
    onClick={[Function]}
  >
    WA
  </a>
</div>
`;
